app/views/account/shared/_header.html.slim

```diff
@@ -0,0 +1,95 @@
+nav.header-navbar.navbar-expand-md.navbar.navbar-with-menu.navbar-static-top.navbar-dark.bg-gradient-x-grey-blue.navbar-border.navbar-brand-center
+  .navbar-wrapper
+    .navbar-header
+      ul.nav.navbar-nav.flex-row
+        li.nav-item.mobile-menu.d-md-none.mr-auto
+          a.nav-link.nav-menu-main.menu-toggle.hidden-xs[href="#"]
+            i.ft-menu.font-large-1
+        li.nav-item
+          = link_to(root_path, class: "navbar-brand") do
+            = image_tag('stack-logo-light.png', alt: 'logo', class: 'brand-logo')
+            h2.brand-text
+              - if current_user.admin?
+                | ADMIN PANEL
```

Comment:
remove this. 
Sales Assistant always
+ merge with master and put logo here (we already have this)
only logo, no text

Reply:
Done. Screenshot updated